lib/main.js

```diff
@@ -23,7 +23,11 @@ export default class PackageDeps{
       }
     })
     if (packagesToInstall.length) {
-      const APMPath = Path.join(Path.dirname(process.execPath), 'resources', 'app', 'apm', 'bin', 'apm')
+      let APMPath = Path.dirname(process.execPath)
+      if (process.platform === 'darwin') {
+        APMPath = Path.join(APMPath, '..', '..')
```

Comment:
This works:

`Path.join(APMPath, '..', '..', '..', '..', 'Resources', 'app', 'apm', 'bin', 'apm')`